discretisedfield/mesh.py

```diff
@@ -229,6 +238,38 @@ def __init__(
         if "isplane" not in attributes.keys():
             self.attributes["isplane"] = False
 
+    @property
+    def bc(self):
+        """Boundary condition for the mesh."""
+        return self._bc
+
+    @bc.setter
+    def bc(self, bc):
+        if bc not in {"neumann", "dirichlet", ""}:
+            for char in bc:
+                if char not in self.region.dims:
+                    raise ValueError(f"Axis {char} is absent in {self.region.dims}.")
+                elif bc.count(char) > 1:
+                    raise ValueError(f"Axis {char} is present more than once.")
+            self._bc = bc
+        else:
+            self._bc = bc
+
+    @property
+    def cell(self):
+        """The cell dimensions of the mesh."""
+        return self._cell
```

Comment:
This should be computed on-the-fly. Mesh should only save `n` (to simplify scaling).